docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.3"
+services:
+  postgresdb:
+    image: postgres:11.10
+    restart: on-failure
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_PASSWORD=postgres
```

Comment:
Is this a real password? Same question as before.

Reply:
This is a real password we're using for testing only, that's the password our container starts with

Reply:
Next question: Will the password stick in the database or it is only during tests?

Reply:
It's only during tests, for test phase we startup the PostgreSQL container to make sure that flyway migration doesn't fail, since H2 doesn't support all the features PostgreSQL does and after tests we kill this container, so once the build is done, no relationship stays with this database